### DISCUSSION TITLE
The VS Code extension does not work

### DISCUSSION THREAD

**Discussion by antonsxh** [community/community#73467] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I don't know how to use the extension.
I watched a tutorial where it says to press Ctrl + Shift + P and then type >codespaces connect. I tried that and every command from GitHub Codespaces does not work and it says this COmmand was not found.

**Comment by Seol-JY** [comment]
Have you installed GitHub Codespaces Extension?

**Reply by antonsxh** [reply]
Yes, I have installed the extension.

**Reply by Seol-JY** [reply]
@snasyde 
<img width="591" alt="스크린샷 2023-11-06 오전 2 41 38" src="https://github.com/community/community/assets/70826982/8f590ce5-b2a1-4a93-bf16-0d0beaa06566">
I can see. Is it true that it is not set to disabled?

**Reply by antonsxh** [reply]
I see it too, but when I click on it, I get an error
![image](https://github.com/community/community/assets/96841334/9864997c-4a29-437e-b0a6-d588b05dc17f)
![image](https://github.com/community/community/assets/96841334/3645fe74-9212-44b9-9fe6-dc94b6858e92)

**Reply by Seol-JY** [reply]
@snasyde 

[GitHub Codespaces extension for Visual Studio Code cannot connect](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-your-connection-to-github-codespaces#github-codespaces-extension-for-visual-studio-code-cannot-connect)
If you cannot connect to a codespace from Visual Studio Code desktop, use the following troubleshooting steps.

Check that you have the latest version of the GitHub Codespaces extension installed. The extension is a preview release and frequent updates are released.
In Visual Studio Code, display the "Extensions" tab.
Select the GitHub Codespaces extension to display the extension's overview page.
If an update is available, a button is shown, click Update to X.X.X to upgrade to the latest version.
Check whether you are using the stable build of Visual Studio Code or the [Visual Studio Code Insiders](https://code.visualstudio.com/insiders/) release (nightly updates). If you are using the insiders release, try installing the [stable build](https://code.visualstudio.com/).
Your company network may be blocking the connection. If possible, check any logging for rejected connections on your device.
If you still cannot connect, you may need to contact support. For more information, see "[Working with support for GitHub Codespaces](https://docs.github.com/en/codespaces/troubleshooting/working-with-support-for-github-codespaces)."

https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-your-connection-to-github-codespaces

**Reply by antonsxh** [reply]
I have installed the latest version and to my knowledge my internet does not block GitHub Codespaces
![image](https://github.com/community/community/assets/96841334/112f87a3-eba0-472d-8ff7-a875c74f4b51)

**Reply by Seol-JY** [reply, marked as answer]
@snasyde I've tried to reproduce the problem and searched a lot, but I don't know anything about the problem anymore. I think reinstalling the extension or reinstalling vscode should be a last resort. Sorry for not being of help.

**Reply by antonsxh** [reply]
Okay, but thanks for the help anyway

**Reply by antonsxh** [reply]
I uninstalled everything that has to do with programming and now it works lol
